Sources/SmithyRetries/RetryErrorType+CRT.swift

```diff
@@ -0,0 +1,21 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import enum SmithyRetriesAPI.RetryErrorType
+import AwsCommonRuntimeKit
+
+public extension RetryErrorType {
```

Comment:
This extension was split from the declaration of `RetryErrorType`, which was placed in SmithyRetriesAPI.

This extension goes into the implementation module SmithyRetries since it is dependent on CRT.